tests/v2/validation/pipeline/Jenkinsfile.recurring

```
@@ -37,12 +37,10 @@ node {
       corralBranch = "${env.RANCHER_CORRAL_PACKAGES_REPO_BRANCH}"
     }
 
-    def rancherRepo = scm.getUserRemoteConfigs()[0].getUrl()
     if ("${env.REPO}" != "null" && "${env.REPO}" != "") {
       rancherRepo = "${env.REPO}"
     }
 
-    def corralRepo = scm.getUserRemoteConfigs()[1].getUrl()
     if ("${env.RANCHER_CORRAL_PACKAGES_REPO_URL}" != "null" && "${env.RANCHER_CORRAL_PACKAGES_REPO_URL}" != "") {
       corralRepo = "${env.RANCHER_CORRAL_PACKAGES_REPO_URL}"
```

should this be 
```suggestion
      def corralRepo = "${env.RANCHER_CORRAL_PACKAGES_REPO_URL}"
```
as corralRepo is not defined anymore after line 45.. similarly for line 42 rancherRepo is not defined anymore.

---

It's cause of the other PR I sent offline @anupama2501. 

@igomez06 we don't need this anymore, we treat repos as environment vars and there's safety net checks right after these (ex L46), so repos are env vars now